graphdatascience/graph_data_science.py

```diff
@@ -37,10 +42,18 @@ def __init__(self, endpoint: Union[str, QueryRunner], auth: Any = None, aura_ds:
         else:
             self._query_runner = endpoint
 
-        super().__init__(self._query_runner, "gds")
+        try:
+            server_version_string = self._query_runner.run_query("RETURN gds.version()").squeeze()
+        except Exception as e:
+            raise UnableToConnectError(e)
+
+        server_version_numbers = map(int, server_version_string[:5].split("."))
```

Comment:
```suggestion
        foo = server_version_string.split(".")[:3]
        server_version_numbers = map(lambda s: int(re.sub("-alpha[\d]+", "", s)), foo)
```

also import re, and check that it works